diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 3.0)
+cmake_minimum_required (VERSION 3.5)
 
 if (POLICY CMP0042)
   cmake_policy (SET CMP0042 NEW)
@@ -8,7 +8,7 @@
   cmake_policy (SET CMP0063 NEW)
 endif (POLICY CMP0063)
 
-project(glog VERSION 0.5.0 LANGUAGES CXX)
+project(glog VERSION 0.5.0 LANGUAGES C CXX)
 
 set (CPACK_PACKAGE_NAME glog)
 set (CPACK_PACKAGE_DESCRIPTION_SUMMARY "Google logging library")
@@ -17,7 +17,7 @@
 set (CPACK_PACKAGE_VERSION_PATCH ${PROJECT_VERSION_PATCH})
 set (CPACK_PACKAGE_VERSION ${PROJECT_VERSION})
 
-option (BUILD_SHARED_LIBS "Build shared libraries" ON)
+option (BUILD_STATIC_LIBS "Build shared libraries" ON)
 option (PRINT_UNSYMBOLIZED_STACK_TRACES
   "Print file offsets in traces instead of symbolizing" OFF)
 option (WITH_CUSTOM_PREFIX "Enable support for user-generated message prefixes" OFF)
@@ -802,12 +802,12 @@
     FIXTURES_REQUIRED "cmake_package_config;cmake_package_config_working")
 endif (BUILD_TESTING)
 
-install (TARGETS glog
-  EXPORT glog-targets
-  RUNTIME DESTINATION ${_glog_CMake_BINDIR}
-  PUBLIC_HEADER DESTINATION ${_glog_CMake_INCLUDE_DIR}/glog
-  LIBRARY DESTINATION ${_glog_CMake_LIBDIR}
-  ARCHIVE DESTINATION ${_glog_CMake_LIBDIR})
+#install (TARGETS glog
+#  EXPORT glog-targets
+#  RUNTIME DESTINATION ${_glog_CMake_BINDIR}
+#  PUBLIC_HEADER DESTINATION ${_glog_CMake_INCLUDE_DIR}/glog
+#  LIBRARY DESTINATION ${_glog_CMake_LIBDIR}
+#  ARCHIVE DESTINATION ${_glog_CMake_LIBDIR})
 
 if (WITH_PKGCONFIG)
   install (
@@ -840,8 +840,8 @@
   ${CMAKE_CURRENT_BINARY_DIR}/glog-config-version.cmake
   COMPATIBILITY SameMajorVersion)
 
-export (TARGETS glog NAMESPACE glog:: FILE glog-targets.cmake)
-export (PACKAGE glog)
+#export (TARGETS glog NAMESPACE glog:: FILE glog-targets.cmake)
+#export (PACKAGE glog)
 
 get_filename_component (_PREFIX "${CMAKE_INSTALL_PREFIX}" ABSOLUTE)
 
@@ -885,5 +885,5 @@
   FILES_MATCHING PATTERN "*.cmake"
 )
 
-install (EXPORT glog-targets NAMESPACE glog:: DESTINATION
-  ${_glog_CMake_INSTALLDIR})
+#install (EXPORT glog-targets NAMESPACE glog:: DESTINATION
+#  ${_glog_CMake_INSTALLDIR})
